paddlenlp/transformers/artist/modeling.py

```diff
@@ -143,12 +51,12 @@ class ArtistForConditionalGeneration(GPTLMHeadModel):
 
     """
 
-    pretrained_init_configuration = pretrained_init_configuration
-    pretrained_resource_files_map = pretrained_resource_files_map
+    pretrained_init_configuration = Artist_PRETRAINED_INIT_CONFIGURATION
+    pretrained_resource_files_map = Artist_PRETRAINED_RESOURCE_FILES_MAP
 
-    def __init__(self, gpt):
-        super().__init__(gpt)
-        self.lm_head = GPTLMHead(self.gpt.config["hidden_size"], self.gpt.config["vocab_size"])
+    def __init__(self, config: ArtistConfig):
+        super().__init__(config)
+        self.lm_head = GPTLMHead(config["hidden_size"], config["vocab_size"])
```

Comment:
```suggestion
        self.lm_head = GPTLMHead(config.hidden_size, config.vocab_size)
```

建议通过属性的方式来获取，并不是通过键值对。